src/Tribe/Integrations/WPML/WPML.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+
+/**
+ * Class Tribe__Events__Integrations__WPML__WPML
+ *
+ * Handles anything relating to The Events Calendar and WPML integration
+ *
+ * This class is meant to be an entry point hooking specialized classes and not
+ * a logic hub per se.
+ */
+class Tribe__Events__Integrations__WPML__WPML {
```

Comment:
The case here is that we might end up with more integrations and having the main file of each called `Main` would make skimming the code quite a chore.
We use this convention elsewhere in the code where we call the main class `Tribe_..._Some_Some`